Mage/src/main/java/mage/abilities/effects/common/counter/AddCountersAllEffect.java

```diff
@@ -50,6 +72,14 @@ public boolean apply(Game game, Ability source) {
         return false;
     }
 
+    @Override
+    public String getText(Mode mode) {
+        if (!staticText.isEmpty()) {
+            return staticText;
+        }
+        return CardUtil.getAddRemoveCountersText(amount, counter, getTargetPointer().describeTargets(mode.getTargets(), "that creature"), true);
```

Comment:
This can't be right. This isn't a targeted effect. I'm going to revert this back to the staticText you removed for now, and fix for the amount later.